charts/jenkins-x-crds/templates/pipelineactivities-crd.yaml

```diff
@@ -1,23 +1,16 @@
+---
 {{- if .Capabilities.APIVersions.Has "apiextensions.k8s.io/v1beta1" }}
```

Comment:
Not very sure, if we should keep support for v1beta1 as v1 api for crd is available since 1.16, and no cloud provider supports kubernetes versions lower than 1.16 these days.
It's probably best to remove it imo, wdyt @rawlingsj ?

Reply:
It might be getting more feedback from Jenkins X users to ask what Kubernetes versions they are running.  Also for when releasing ensuring we add a note here https://jenkins-x.io/v3/about/changes/.

The other thing that we should check is an upgrade works as expected rather than just new installs.

Totally agree that cleaning up old version cruft is good, it's just that it brings risk too so it's good to ensure existing installs continue to work as expected.

Reply:
> it's just that it brings risk too so it's good to ensure existing installs continue to work as expected.

I tested the upgrade on my local k3s cluster (that's why I had to add `preserveUnknownFields: false`  - added a note in the readme), I use AWS in production, and they still dont have 1.22 (GCP and Azure have though I think).
I have asked in the the user channel if anyone is using < 1.16 atm.

Reply:
ok that all sounds great

Reply:
Also asked @

> one last thing, I noticed this today and wondered if it affect the work in this PR as on first glance it mentions similar fields https://mail.google.com/mail/u/0/#search/kubernetes/FMfcgzGmtXMhWvHJbHMTPdwXPJpcVqKh

Wont load in gmail ... but is it abt the 1.23 bug with crds?